Content.IntegrationTests/Tests/Localization/LocalizedDatasetPrototypeTest.cs

```diff
@@ -0,0 +1,34 @@
+using Content.Shared.Dataset;
+using Robust.Shared.Localization;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests.Localization;
+
+[TestFixture]
+public sealed class LocalizedDatasetPrototypeTest
+{
+    [Test]
+    public async Task ValidProtoIdsTest()
+    {
+        await using var pair = await PoolManager.GetServerClient();
+
+        var server = pair.Server;
+        var protoMan = server.ResolveDependency<IPrototypeManager>();
+        var localizationMan = server.ResolveDependency<ILocalizationManager>();
+
+        var protos = protoMan.EnumeratePrototypes<LocalizedDatasetPrototype>();
+
```

Comment:
i think you need to sort them so that the test is always the same (IIRC EnumeratePrototypes doesnt guarantee a specific order)

Reply:
Sure, easy enough.